### DISCUSSION TITLE
[Bug]: Don't get an e-mail when the form is filled in!

### DISCUSSION THREAD

**Discussion by mertcanfirat** [pimcore/pimcore#17163] [Q&A / Support]
### Pimcore version

6.7.2

### Steps to reproduce

Hello, when the contact form on our site is filled out, we do not receive information that 1 form has been filled out on our site and therefore we have to look through the administration panel every time.

I checked the SMTP settings, I checked the form, there is no problem.

### Actual Behavior

When the contact form on our site is filled out, we do not receive information that 1 form has been filled out on our site

### Expected Behavior

Is it possible for the form to inform us via e-mail when the form is filled out from our site?

**Comment by kingjia90** [comment]
Converted to discussion as Pimcore 6 is EOL and we don't support it anymore.

However, which bundle are you using to make the contact form? I recall there's no out-of-the-box solution and had to use something like [DACHCOM](https://pimcore.com/en/developers/marketplace/dachcom-digital/formbuilder_e5462) one on my very first project
